### ISSUE TITLE
PUT body no longer set.

### ISSUE THREAD

**Issue by jcao219** [ember-cli/ember-fetch#86]
Up until v3.4.1, mungOptionsForFetch sets the PUT body correctly.

Example failing test case:
```javascript
  const baseOptions = {
    url: '/',
    method: 'PUT',
    data: { a: 1 }
  };

  let options = mungOptionsForFetch(baseOptions, this.basicAdapter);
  assert.equal(options.body, JSON.stringify(baseOptions.data), 'PUT request body correctly set');
```

**Comment by rwjblue** [comment]
@nlfurniss - Seems related to https://github.com/ember-cli/ember-fetch/pull/84?

**Comment by nlfurniss** [comment]
@jcao219 fixed in 3.4.3
